SLAM_cone_msg[] cones_
float32 x_global_vehicle
float32 y_global_vehicle
float32 theta_global_vehicle
time map_time
bool closed_
